internal/bgp/bgpfrr/parse.go

```diff
@@ -24,10 +26,16 @@ type Route struct {
 const bgpConnected = "Established"
 
 type FRRNeighbor struct {
-	RemoteAs   int    `json:"remoteAs"`
-	LocalAs    int    `json:"localAs"`
-	BgpVersion int    `json:"bgpVersion"`
-	BgpState   string `json:"bgpState"`
+	RemoteAs     int    `json:"remoteAs"`
+	LocalAs      int    `json:"localAs"`
+	BgpVersion   int    `json:"bgpVersion"`
+	BgpState     string `json:"bgpState"`
+	MessageStats struct {
+		UpdatesSent int `json:"updatesSent"`
+	} `json:"messageStats"`
```

Comment:
This change seems unrelated to the commit message?

Reply:
moved to a separate commit and updated unit tests, thanks